README.md

```diff
@@ -159,6 +159,28 @@ It is also possible to determine whether a layer supports a specific ApiType:
   };
 ```
 
+Gain and gamma effects are supported only when the blob is retrieved, because the effects are applied in the library. 
+
+```javascript
+  const getMapParamsWithGainAndGamma = {
+    bbox: bbox,
+    fromTime: new Date(Date.UTC(2018, 11 - 1, 22, 0, 0, 0)),
+    toTime: new Date(Date.UTC(2018, 12 - 1, 22, 23, 59, 59)),
+    width: 512,
+    height: 512,
+    format: MimeTypes.JPEG,
+    gain: 1.2,
+    gamma: 0.9
+  };
+  const imageBlob = await layer.getMap(getMapParamsWithGainAndGamma, ApiType.WMS);
+  const imageBlob2 = await layer.getMap(getMapParamsWithGainAndGamma, ApiType.PROCESSING);
+```
+
+As mentioned above, both of the images above should be _exactly_ the same, no matter which API type is used. 
+This is still true when applying gain and gamma effects.
+
+When retrieving an image URL with gain and gamma applied, an error is thrown, because the retrieved URL points directly to the image on the services with no applied effects.
```

Comment:
This is good, I would just suggest adding functino name to make it clearer what we are talking about:
```
When retrieving an image URL (`getMapUrl`) with gain and gamma applied, an error is thrown, because the retrieved URL points directly to the image on the services with no applied effects.
```

Reply:
Thanks, added the function name.

Reply:
Resolving so I can merge